<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Database</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dbNameCombo"/>
          </item>
          <item>
           <widget class="QPushButton" name="getDatabasesButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Get databases</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="collectionsTitle">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Collections</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="usersCollectionCombo"/>
              </item>
              <item>
               <widget class="QPushButton" name="getCollectionsButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Get Collections</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="authorizeButton">
            <property name="text">
             <string>Authorize</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="statusLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="curStatus">
              <property name="text">
               <string>Offline</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="userList">
            <property name="text">
             <string>Users List</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="usersList"/>
          </item>
          <item>
           <widget class="QPushButton" name="getUsersButton">
            <property name="text">
             <string>Get users</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addUserButton">
            <property name="text">
             <string>Add user</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteUserButton">
            <property name="text">
             <string>Delete user</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="infoLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="usetInfo">
            <property name="text">
             <string>user Info</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="userInfoLayout">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="name">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="userNameEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="surname">
                <property name="text">
                 <string>Surname</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="userSurnameEdit"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="email">
                <property name="text">
                 <string>Email</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="userEmailEdit"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="nickname">
                <property name="text">
                 <string>Nickname(unique)</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="userNickNameEdit"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="password">
                <property name="text">
                 <string>Password</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="userPasswordEdit"/>
              </item>
              <item row="5" column="1">
               <widget class="QCheckBox" name="deleteUserBox">
                <property name="text">
                 <string>deleted user</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="saveUserInfoButton">
              <property name="text">
               <string>Save Data</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLogIn"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogIn">
   <property name="text">
    <string>LogIn</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
